pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -469,41 +468,37 @@ func (cr *PerconaServerMongoDB) OwnerRef(scheme *runtime.Scheme) (metav1.OwnerRe
 // The new (semver-matching) version is determined either by the CR's API version or an API version specified via the CR's annotations.
 // If the CR's API version is an empty string, it returns "v1"
 func (cr *PerconaServerMongoDB) setVersion() error {
-	apiVersion := cr.APIVersion
+	if len(cr.Spec.CRVersion) > 0 {
+		return nil
+	}
```

Comment:
What if user will want to apply configuration with new CR version?

P.S. I'm just interested, not sure it's valid case.

Reply:
Ahh, i forget to add example to the cr.yaml =)
But basically, you can change this value from yaml, according to the update docs